spec/javascripts/application_spec.js

```diff
@@ -0,0 +1,10 @@
+
+describe('header', function() {
+  beforeEach(function() {
+    MagicLamp.load("layouts/header");
+  });
+
+  it ('has the right avatar src', function() {
+    expect($("a.dropdown-toggle img").attr("src")).to.equal("https://github.com/foo.png?size=40");
```

Comment:
It's definitely not going to make it out of WIP. This is the first thing I saw on the page when I was trying out how Magic Lamp works, and I just kept it as a placeholder until I write some real tests.